dht/sharder.hh

```diff
@@ -28,28 +28,58 @@
 
 namespace dht {
 
+
+// Utilities for sharding ring partition_range:s
```

Comment:
What's this ':'? Is it some sort of convention?

Reply:
I learned it from @elcallio so I don't know the origin story. It's useful to keep an identifer seprated from the plural marker, so it's clear the s isn't part of it, in case you want to search for it.